devito/ops/transformer.py

```diff
@@ -1,3 +1,29 @@
+from devito.ir.iet import Callable, Expression, FindNodes
+from devito.ops.node_factory import OPSNodeFactory
+
+
+def opsit(trees, count):
+    node_factory = OPSNodeFactory()
+    expressions = []
+    for tree in trees:
+        expressions.extend([
+            Expression(make_ops_ast(i.expr, node_factory))
+            for i in FindNodes(Expression).visit(tree.inner)
+        ])
+
+    arguments = []
+    for exp in expressions:
+        func = [f for f in exp.functions if f.name != "OPS_ACC_size"]
+        for f in func:
+            f.is_OPS = True
```

Comment:
all our objects are immutable (and yes, our bad, we should enforce it...), so you can really change the value of these flags dynamically.

Reply:
Ok - I'll create new ones with OPS types, is that right?

Reply:
correct